app/code/Magento/InventoryInStorePickupAdminUi/README.md

```diff
@@ -0,0 +1,24 @@
+# InventoryInStorePickupAdminUi module
+
+The `InventoryInStorePickupAdminUi` module provides Inventory Management service contracts. 
```

Comment:
Should be rephrased to something like "...  extends the Magento Admin UI to add Inventory In Store Pickup functionality."